client/lib/DomExtender.ts

```diff
@@ -95,14 +111,58 @@ const ExtensionFns = {
       nodePointer,
     );
   },
-  async $getComputedVisibility(): Promise<INodeVisibility> {
+  async $clearValue(): Promise<void> {
```

Comment:
This only works in a text field, right? Wondering if this is named in a way you would think it would just clear out the "Value" if you used it on a radio or option

Reply:
Do you have another suggestion?

My thought is that none of these are set in stone. We're writing scripts and experimenting.

Reply:
Seems like clearText might make more sense